slidez-fe-core/src/containers/poll-editor/PollEditor.scss

```diff
@@ -1,47 +1,86 @@
 @import "src/global/global";
 
 .app {
-    margin: 2rem;
+    margin: 0.5rem 1rem 1rem 1rem;
 }
 
-.text-input,
-.poll-name {
-    margin-bottom: 20px;
+.text-input {
+    font-family: "Rubik";
```

Comment:
We apply font-family globally, so or make so in .app class or remove, as it should be Rubik by default

Reply:
Removed